src/model/ScheduleFile.hpp

```diff
@@ -58,6 +58,8 @@ namespace model {
 
     ScheduleFile(const ExternalFile& externalfile, int column = 1, int rowsToSkip = 0);
 
+    ScheduleFile(const Model& model, const std::string& filename, int column = 1, int rowsToSkip = 0);
```

Comment:
```suggestion
    ScheduleFile(const Model& model, const openstudio::path& filePath, int column = 1, int rowsToSkip = 0);
```

Reply:
Like I said on https://github.com/NREL/OpenStudio/issues/4763#issuecomment-1502822755, I think ExternalFile::getExternalFile should have taken a path to begin with, but here this is new so I'd say let's go with path.

Reply:
Oh I see. You're advocating for a potential API break on `getExternalFile`. Hmm...

Reply:
Could we deprecate `getExternalFile` with `filename` argument in favor of a new one with `filepath` argument?